datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxController.java

```diff
@@ -184,6 +185,13 @@ public ObjectStorageValidateResponse validateCloudStorage(@ValidStackNameFormat
                 sdxValidateCloudStorageRequest.getDataAccessRole(), sdxValidateCloudStorageRequest.getRangerAuditRole());
     }
 
+    @Override
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.DESCRIBE_DATALAKE)
+    public ValidationResult validateBackupStorage(@ValidStackNameFormat @ValidStackNameLength String clusterName) {
```

Comment:
Could you please cover this one as well with UT?

Reply:
done